projects/layout/components/card/card.styles.less

```diff
@@ -25,6 +25,10 @@
     > * {
         .scrollbar-hidden();
     }
+
+    :where(a&) {
+        color: inherit;
```

Comment:
We should just set `color: var(--tui-text-primary)` all the time, no matter what tag, since colors that come from appearances have higher specificity anyway, due to double attribute selector `[tuiAppearance][data-appearance='xxx']`

Reply:
Oops, I see, `neutral` appearance does not set color. It's a bug, all status appearances should have `color: var(--tui-text-primary)` here:
https://github.com/taiga-family/taiga-ui/blob/main/projects/core/styles/theme/appearance/status.less#L11